### PR TITLE
Docs : Add a yml and docker command when m1 /  self-host/index install

### PR DESCRIPTION
**Pull request by 0r0loo**
### Features and Changes
#1188 
Added modifications for M1 users to run docker images in their local environment

### Dependencies
N/A
### Testing
N/A
### Screenshots
N/A

### CODE REVIEW

File: docs/docs/self-host/index.mdx [hunk 1]
```diff
@@ -46,6 +46,41 @@ volumes:
...
+    ports:
+      - "3000:3000"
+      - "3100:3100"
+    depends_on:
+      - mongo
+    environment:
+      - MONGODB_URI=mongodb://root:password@mongo:27017/
+    volumes:
+      - uploads:/usr/local/src/app/packages/back-end/uploads
+    platform: linux/x86_64
```
Comment (tinahollygb) [inline docs/docs/self-host/index.mdx:72]: Did you mean to change this to `linux/amd64`? This one and the value on line 60 appear to be a different value than the inline command you've specified.

Comment (0r0loo) [reply docs/docs/self-host/index.mdx:72]: Oh Sorry, Mistake🥲

**Review by tinahollygb — APPROVED** [review]
Another user had this same issue on their M1. They tried the `DOCKER_DEFAULT_PLATFORM` suggestion and that worked so this seems to be the right fix to get it to work on M1's. They also mentioned they needed to downgrade their version of MongoDB:

> Thanks for the help. Got it to work after adding DOCKER_DEFAULT_PLATFORM and using an earlier version of MongoDB 5.0+ as "MongoDB 5.0+ requires a CPU with AVX support, and your current system does not appear to have that!"

@0r0loo which version of MongoDB are you using? Did you run into issues with the latest Mongo?

You can see that conversation here: https://growthbookusers.slack.com/archives/C01T6PKD9C3/p1681923484553939

**Comment by 0r0loo** [comment]
Yes, I used the latest MongoDB version. 6.0.5

**Pull request merged**